src/components/dashboard/dashboardItem/ItemBase.js

```diff
@@ -212,7 +212,7 @@ export const DashboardVideoItem = ({ item }) => {
         >
             <ReactPlayer
                 url={item.getLinkTarget()}
-                light={true}
+                light={item.content.thumbnailUrl || true}
```

Comment:
This change means that if the data that comes from dashboard-aggregator includes a `thumbnailUrl` (as I added manually to one of the videos in the data.json below) then noembed won't be called at all.